src/pfb/exporters/tsv.py

```diff
@@ -103,6 +111,9 @@ def _to_tsv(reader, dir_path, handlers_by_name):
             parent_node = r["dst_name"]
             parent_id = r["dst_id"]
 
+            if parent_node in plural_parents:
```

Comment:
this is an unnecessary, nit-picky, fancy, one-liner suggestion. But honestly I think your way is more readable

```python
parent_node = plural_parents.get(parent_node, parent_node)
```

Reply:
I think we should keep it readable. If people are going to work with PFBs I think what was written is sufficient. Both lines are O(1) anyways.